src/OrderFormAcceptanceTests.Steps/Utils/EmailUtils.cs

```diff
@@ -8,16 +8,8 @@
 namespace OrderFormAcceptanceTests.Steps
 {
     [Binding]
-    class EmailUtils
+    internal class EmailUtils
```

Comment:
As you've removed a few bits, you could set this class as `internal static class EmailUtils`